pkg/queue/health/probe_test.go

```diff
@@ -41,7 +41,7 @@ func TestTCPProbe(t *testing.T) {
 	}
 	// Connecting to the server should work
 	if err := TCPProbe(config); err != nil {
-		t.Errorf("Expected probe to succeed but it failed with %v", err)
+		t.Errorf("Expected probe to succeed but failed with error %v", err)
```

Comment:
Since you're changing this, just `Probe failed with: %v`, is probably sufficient.